Sources/ComposableArchitecture/Store.swift

```diff
@@ -322,32 +322,17 @@ public final class Store<State, Action> {
     action fromLocalAction: @escaping (LocalAction) -> Action
   ) -> AnyPublisher<Store<LocalState, LocalAction>, Never>
   where P.Output == LocalState, P.Failure == Never {
-
```

Comment:
What are the chances of someone doing something like `.publisherScope(state: { $0.collect(2).compactMap(\.last) })`? Do people use this or is it mainly for `.ifLet`